tcp-to-blob/README.md

```diff
@@ -104,23 +92,60 @@ Optional:
 Recommend creating a `tcp-to-blob/.env/env-${stage}.sh` to set these and re-load env as needed without risking
 committing them to version control.
 
-## Deploy environment to Azure Kubernetes Service (AKS)
+## Deploy TCP to  BLOB and related resources
+
+### Deploy using docker
+We have prepared a docker file, `tcp-to-blob/deploy/Dockerfile_deployer`, with all prerequisites needed for deploying.
+
+#### Prerequisites
+
+- [Docker](https://command-not-found.com/docker)
+
+#### Procedure
 
 1. `git clone https://github.com/Azure/azure-orbital-integration.git`
-2. If using [Azure Cloud Shell](https://learn.microsoft.com/en-us/azure/cloud-shell/overview), you are already logged in. Otherwise, Ensure [logged in](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli) to Azure CLI and default
+2. `docker build . -f deploy/Dockerfile_deployer -t orbital-integration-deployer`
+3. `NAME_PREFIX=<desired_name_prefix>` Set prefix for names of resources to be deployed.
+4. `docker run -it -e NAME_PREFIX orbital-integration-deployer:latest`
+5. The command above will bring you to a container shell. In container shell:
+   1. `az login`
+   2. `az account set -s <your_subscription>`
+   3. `git pull`
+   4. (optional) Update .env/env-template.sh if desired. See Environment Variables section above.
+   5. `./tcp-to-blob/deploy/deploy-in-docker.sh`
+   6. If any commands ask to install extensions, type "y".
```

Comment:
Not required if we install `az extension add --name portal` during docker build.